mackerel/data_source_mackerel_alert_group_setting_test.go

```diff
@@ -4,8 +4,8 @@ import (
 	"fmt"
 	"testing"
 
-	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/acctest"
-	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/resource"
+	"github.com/hashicorp/terraform-plugin-testing/helper/acctest"
+	"github.com/hashicorp/terraform-plugin-testing/helper/resource"
```

Comment:
The test helpers provided by terraform-plugin-sdk and terraform-plugin-testing are the same, but they cannot coexist.